### PR TITLE
[AR] 05-1

### PR DESCRIPTION
**Pull request by WejdanAlOtaibi**

### CODE REVIEW

**Comment by AliEl-01** [comment]
Hi Wejdan, 
Please edit the name of your PR from **05-1** to** [AR] 05-1.md** or** [AR] Translation of 05-1.md** ... just to know that this is an Arabic one! 
TnQ

File: docs/ar/week05/05-1.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+
+
+المدخل $\gamma$ في المعادلة التحديثية التكرارية يسمى **مقياس الخطوة**. بشكل عام نحن لا نعرف القيمة المثالية لمقياس الخطوة ولذلك سنضطر تجربة قيم مختلفة.
+المعيار القياسي سيكون تجربة مجموعة من القيم على ميزان للأداء ونختار أفضل قيمة بناء على الميزان. هنالك سناريوهات مختلفة قد تحدثز
+الصورة بالأعلى تصف هذه السناريوهات بالنسبة 1D quadratic.إذا معدل التعلم منخفض للغاية إذن سنضطر لخلق تقدم ثابت للوصول إلى الحد الأدنى.ولكن هذه  الخطوة ستأخذ وقت اطول من الوقت المثالي.
+صعب جدا (وشبه مستحيل ) أن نحصل على مقياس خطوة يصلنا بشكل مباشر للحد الأدنى. ما سنريده بشكل مثالي هو أن نحصل على مقياس خطوة اكبر بقليل من القياس المثالي. هذا ما سعطينا اسرع تقارب (convergence). ولكن إذا استعملنا معدل تعلم كبير التكرار سيصبح أبعد وأبعد من الحد الأدنى وسنحصل على تباعد (divergence). نريد أن نستعمل معدل تعلم قليل البعد (less diverging).
+
+<center>
+<img src="{{site.baseurl}}/images/week05/05-1/step-size.png" style="zoom: 70%; background-color:#DCDCDC;" /><br>
+<b>Figure 1:</b> مقياس خطوات 1D Quadratic
```
Comment (AliEl-01) [inline docs/ar/week05/05-1.md:46]: Hi, 
Please if you can translate the terme ^Figure^ into ^الشكل^. That's will be better.
TnQ

File: docs/ar/week05/05-1.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+بشكل مختصر فوائد النزول التدريجي العشوائي هي:
+ 1. هنالك الكثير من المعلومات المتكررة في جميع الـinstances. الـ(SGD) يمنع الكثير من تلك العمليات المكررة.
+ 2. في المراحل الابتدائية الازعاج يعتبر بكميات قليلة مقارنة بالمعلومات في التدرج. وهكذا خطوة الـ(SGD) جيدة مثل خطوة الـGD ولكن بشكل افتراضي
+ 3. *التليين* - الازعاج في تحديث SGD يستطيع أن يمنع التقارب لقيمة صغرى محلية سيئة (سطحية)
+ 4. النزول التدريجي العشوائي تكلفة تشغيله تعتبر منخفضة (لأنك لن تتجاوز جميع النقاط البيانات).
+
+
+### (Mini-batching) التحزيم الصغير 
+
+في التحزيم الصغير سنضع بعين الاعتبار الخسارة على العديد من instances المختارة بشكل عشوائي بدلا من حسابها على instance واحد فقط.
```
Comment (AliEl-01) [inline docs/ar/week05/05-1.md:98]: Actually I don't know why the term ^instance^ is not translated !!!

**Review by AliEl-01 — CHANGES_REQUESTED** [review]

**Comment by WejdanAlOtaibi** [comment]
I modified what’s needed, I uploaded the file to the folder, do I need to
send another pull request??

Le dim. 20 sept. 2020 à 3:22 PM, Ali Elfilali <notifications@github.com> a
écrit :

**Comment by WejdanAlOtaibi** [comment]
05-1 [AR]

**Comment by AliEl-01** [comment]
Great! As I see now it's Good!

File: docs/ar/week05/05-1.md [hunk 3]
```diff
@@ -0,0 +1,189 @@
+---
+lang-ref: ch.05-1
```
Comment (Atcold) [inline docs/ar/week05/05-1.md:2]: `lang: ar` missing.

File: docs/ar/week05/05-1.md [hunk 4]
```diff
@@ -0,0 +1,189 @@
+---
+lang-ref: ch.05-1
+title: تقنيات التحسين الجزء الاول
+lecturer: Aaron Defazio
+authors: Vaibhav Gupta, Himani Shah, Gowri Addepalli, Lakshmi Addepalli
+date: 24 Feb 2020
+Translator: Wejdan AlOtaibi
```
Comment (Atcold) [inline docs/ar/week05/05-1.md:7]: Lowercase T, so `translator:`.
`translation-date:` missing as well.

**Review by Atcold — CHANGES_REQUESTED** [review]
Please, edit `docs/_config.yml` as well.

**Review by Atcold — APPROVED** [review]
Fixed these myself.

**Pull request merged**